dashboard/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY . /app
 RUN yarn run prettier-check && yarn run ts-compile-check
 RUN yarn run build
 
-FROM bitnami/nginx:1.22.0-debian-11-r3
+FROM bitnami/nginx:1.23.1-debian-11-r1
```

Comment:
I have confirmed that the update to the branch 1.23.x is also going to be carried out by the Bitnami team near soon.